styles.css

```diff
@@ -146,6 +146,9 @@
 .tasks-modal-priorities label {
     border-radius: var(--input-radius);
     padding: 2px 3px;
+    grid-column: 1;
+    grid-row-start: 1;
+    grid-row-end: 7;
```

Comment:
I have been trying to get used to the new layout, and find the split of 4 values on row 1, and 2 values on row 2, makes it take time to interpret the values every time.

![image](https://github.com/obsidian-tasks-group/obsidian-tasks/assets/4840096/1180c0ca-a08d-4653-9c0b-7393644c54df)

Since there are always at least 2 rows (because never more than 4 priority columns), I think it would be considerably more readable if the initial layout was 3 values on each line.

These would then be:

```
Lowest    Low    Normal
Medium    High   Highest
```

Which I feel would scan quite well.

Reply:
done and you are right, it looks better :)